extensions/sql-database-projects/src/models/project.ts

```diff
@@ -70,6 +73,32 @@ export class Project {
 					this.files.push(this.createProjectEntry(folderElements[f].getAttribute(constants.Include), EntryType.Folder));
 				}
 			}
+
+			// find all pre-deployment scripts to include
+			let preDeployScriptCount: number = 0;
+			const preDeploy = itemGroup.getElementsByTagName(constants.PreDeploy);
+			for (let pre = 0; pre < preDeploy.length; pre++) {
+				this.preDeployScripts.push(this.createProjectEntry(preDeploy[pre].getAttribute(constants.Include), EntryType.File));
+				preDeployScriptCount++;
+			}
+
+			// find all post-deployment scripts to include
+			let postDeployScriptCount: number = 0;
+			const postDeploy = itemGroup.getElementsByTagName(constants.PostDeploy);
+			for (let post = 0; post < postDeploy.length; post++) {
+				this.postDeployScripts.push(this.createProjectEntry(postDeploy[post].getAttribute(constants.Include), EntryType.File));
+				postDeployScriptCount++;
+			}
+
+			if (preDeployScriptCount > 1 || postDeployScriptCount > 1) {
+				window.showInformationMessage(constants.prePostDeployCount);
+			}
+
+			// find all none-deployment scripts to include
+			const noneItems = itemGroup.getElementsByTagName(constants.None);
+			for (let n = 0; n < noneItems.length; n++) {
+				this.nonDeployScripts.push(this.createProjectEntry(noneItems[n].getAttribute(constants.Include), EntryType.File));
```

Comment:
should this be "noneDeployScripts" to match the XML and other variable names?

Reply:
Updated it to "noneDeployScripts" because that actually makes sense to match it.